app/cdap/components/GridTable/components/NestedMenu/types.ts

```diff
@@ -20,8 +20,9 @@ export interface INestedMenuProps {
   columnType: string;
   menuOptions: IMenuItem[];
   title: string;
-  anchorEl: HTMLElement;
-  setAnchorEl: React.Dispatch<React.SetStateAction<HTMLElement>>;
+  anchorEl: HTMLElement[];
+  setAnchorEl: React.Dispatch<React.SetStateAction<HTMLElement[]>>;
   open: boolean;
   handleMenuOpenClose?: (title: string) => void;
+  newIndex?: number;
```

Comment:
what is this newIndex, can we rename it or is this sufficient
maybe newOptionIndex or newCardIndex whatever this particular scenario is expected to get

if it is for general use them we can use item, newMenuItemIndex or newItemIndex

Reply:
It is of no use I've removed